mantidproject/autoreduce
# Scripts can now be selected when re-running

Fixes #173 

This fixes unusual behaviour surrounding which scripts re-runs use.

To test:
* Send a run to autoreduce, wait to finish
* Modify the corresponding reduce.py and reduce_vars.py script
* Open the run previously sent on the WebApp, confirm that the 'Preview Script' shows the old script
* Make some variable changes and submit the re-run
* Confirm the old script is used but with updated variables
* Open the run again, click reset to current script
* Confirm that modifications made in reduce_vars are shown in the corresponding textboxes
* Confirm that the 'Preview Script' shows the new script
* Make some variable changes and submit the re-run
* Confirm the new script is used with updated variables

When testing this ticket be aware of issue #184
